### PR TITLE
Simple Archiver functionality

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Demonstrate:

- BagIt verification from Zip file
- Verifying uploads to S3

### Who is this change for?

📖

### CODE REVIEW

**Comment by kenoir** [comment]
For https://github.com/wellcometrust/platform/issues/2368

File: archive/archiver/src/main/scala/uk/ac/wellcome/platform/archiver/messaging/MessageStream.scala [hunk 1]
```diff
@@ -49,13 +49,20 @@ class MessageStream[T, R] @Inject()(actorSystem: ActorSystem,
       ActorMaterializerSettings(system)
         .withSupervisionStrategy(decider(metricName, Supervision.Resume)))
 
-    val typeConversion = Flow[Message].map(m => fromJson[T](m.getBody).get)
+    val typeConversion = Flow[Message].map( m => {
```
Comment (kenoir) [inline archive/archiver/src/main/scala/uk/ac/wellcome/platform/archiver/messaging/MessageStream.scala]: Can we not use the `log` function from akka streams here?

Comment (mklander) [reply archive/archiver/src/main/scala/uk/ac/wellcome/platform/archiver/messaging/MessageStream.scala]: The Akka log looks a bit all or nothing to me, I wanted to log the incoming msg at `info` and leave everything else at `debug`.

**Comment by kenoir** [comment]
For https://github.com/wellcometrust/platform/issues/2495

**Pull request merged**